README.md

```diff
@@ -76,90 +97,61 @@ in clear text without any encryption (only converted to the Base64 encoding).
-    </ul> Then specify the action: <ul>
-        <li><b>action</b> - action to do</li>
-        <li><b>value</b> (optional) - new value or a parameter of the action</li>
-        <li><b>column</b> (optional) - column id when selecting item in the table</li>
-    </ul>
-</p> Supported actions: <ul>
-    <li><b>press</b> - to press the button</li>
-    <li><b>check</b>|<b>uncheck</b>|<b>toggle</b> - check, uncheck or toggle checkbox</li>
-    <li><b>enter_text</b> - set text in the field, requires <b>value</b> parameter</li>
-    <li><b>switch_radio</b> - activate radio button</li>
-    <li><b>select</b> - select value in the combobox, row in the table or node in the tree, requires <b>value</b> parameter<br />
-        In case of table: select row in the table with given value. If <b>column</b> parameters is not provided, first column will be used. <br />
-        In case of tree: select node in the tree. Use <b>'|'</b> as delimiter for child nodes.</li>
-
-</ul>
-</p>
-<h4>Response</h4>
-<p>JSON format</p>
-<h4>Examples</h4>
-<p>
-    <pre>  # press the "next" button
-curl -X POST 'http://localhost:9999/widgets?id=next&action=press'</pre>
-    <pre>  # set value "test" for the InputField with label "Description"
-curl -X POST 'http://localhost:9999/widgets?label=Description&action=enter_text&value=test'</pre>
-    <pre>  # select row with "test" cell value in the 2-nd column (counting from zero) in table with id "names"
-curl -X POST 'http://localhost:9999/widgets?id=names&action=select&value=test&column=2'</pre>
-    <pre>  # select tree item with in tree with id "files"
-curl -X POST 'http://localhost:9999/widgets?id=files&action=select&value=root|subnode|subnode'</pre>
-</p>
+## Contributing
+
+1. Fork it
+2. Create your feature branch (git checkout -b my-new-feature)
+3. Commit your changes (git commit -am 'Add some feature')
+4. Push to the branch (git push origin my-new-feature)
+5. Create new Pull Request
+
+Please, keep coding style consistent, in case of doubts, please, refer to
+[CppCoreGuidelines](http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines).
+
+
+## Building
+
+In order to build project locally one can use `make`:
+* `make -f Makefile.cvs`
+* `make -C build/ install`
+
+Recommended way is to use `rake`, please refer to
+[libyui Building section](https://github.com/libyui/libyui#building).
+
+Also, see [yast-rake](https://github.com/yast/yast-rake) documentation for
+useful tasks, like `rake version:bump`.
+One can run `rake --tasks` locally to see all available tasks.
+
+## Testing
+
+In order to test changes, one can use yast modules in her system.
+As an alternative, examples from `https://github.com/yast/yast-ycp-ui-bindings`
+project can be used.
+
+For running those examples `yast2-core` package has to be preinstalled.
+
+After `yast-ycp-ui-bindings` project is checked out or `yast2-ycp-ui-bindings-devel`
+package is installed, copy all the files to the `clients` subdirectory anywhere
```

Comment:
@lslezak @jreidinger Hey! I was not sure if there is better way to run examples from that repo. If not, I would propose to create `clients` directory in the examples, so this extra step is not required.